services/metadata/src/metadata/services/avus.clj

```diff
@@ -55,7 +54,7 @@
       (transaction
        (persistence/remove-avu-template-instances template-id [avu-id])
        (persistence/remove-avu avu-id))
-      (throw+ {:error_code error-codes/ERR_DOES_NOT_EXIST
+      (throw+ {:type :clojure-commons.exception/exists
```

Comment:
I guess the only difference is that DOES_NOT_EXIST threw 500, while NOT_FOUND throws a 404. But it's probably more correct for it to throw 404 anyway, so...